Actions/AL-Go-Helper.ps1

```diff
@@ -680,6 +680,12 @@ function ReadSettings {
             $settingsObjects += @($projectWorkflowSettingsObject, $userSettingsObject)
         }
     }
+
+    # If the build is triggered by a pull request the refname will be the merge branch. To apply conditional settings we need to use the base branch
```

Comment:
Minor: `ReadSettings` has a `$branch` parameter.
So, this check is better fitted in Actions\ReadSettings\ReadSettings.ps1

Reply:
Mmh the issue is actually in DetermineProjectsToBuild because that also calls ReadSettings. I could call it from there though? https://github.com/microsoft/AL-Go/blob/e9f4cc63d149d341172cebfc8112e825123683d8/Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.psm1#L158

Reply:
Yeah, I see `AnalyzeProjectDependencies` calls it too. Ok, let's keep it in here.